data/hooks/conf_regen/28-rmilter

```diff
@@ -9,8 +9,6 @@ do_pre_regen() {
 
   install -D -m 644 rmilter.conf \
     "${pending_dir}/etc/rmilter.conf"
-  install -D -m 644 rmilter.socket \
-    "${pending_dir}/etc/systemd/system/rmilter.socket"
```

Comment:
`/etc/systemd/system/rmilter.socket` should be removed from the system if it exists.

On my system I still have this file, coming from previous yunohost versions.
I have :

```
admin@ynh:~$ sudo systemctl restart rmilter.socket                                                       
Job for rmilter.socket failed. See 'systemctl status rmilter.socket' and 'journalctl -xn' for details.   
admin@ynh:~$ systemctl status rmilter.socket                                                             
● rmilter.socket                                                                                         
   Loaded: loaded (/etc/systemd/system/rmilter.socket; enabled)                                          
   Active: inactive (dead) since Wed 2017-01-04 14:30:39 CET; 2min 26s ago                               
   Listen: 127.0.0.1:11000 (Stream)        
```

There is probably something more to do to deregister it from systemd but I don't know the proper incantation (deamon-reload does this ? is it supposed to be done in post_regen hook ?)